lib/content-services/src/lib/document-list/models/node-action.enum.ts

```diff
@@ -0,0 +1,24 @@
+/*!
+ * @license
+ * Copyright 2019 Alfresco Software, Ltd.
```

Comment:
It's not 2019 anymore :)

Reply:
I remember putting 2021 once and the linting or something like that failed.